install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml

```diff
@@ -49,9 +49,13 @@ data:
         imagePullPolicy: {{ .Values.global.imagePullPolicy }}
         securityContext:
           capabilities:
+            drop:
+            - "all"
             add:
-            - NET_ADMIN
-          privileged: true
```

Comment:
I think this was added for OpenShift IIRC.

Reply:
We need a bit more scrutiny when it comes to security flags.